DEVELOPER_GUIDE.md

```diff
@@ -58,6 +62,30 @@
 
 So you want to contribute code to OpenSearch? Excellent! We're glad you're here. Here's what you need to do.
 
+## How this branch will be used
+
+Welcome to feature/identity, this branch is for rapid iteration of identity feature spaces.  Including Authentication and Authorization related features.
+
+Most of the work will be centralized around new systems in the `server` directory in the package `org.opensearch.identity`. As these features are connected into the core systems its expected for this work to branch out into nearly all of the execution paths of the product.
+
+### Who is doing this work?
+
+| Contributor      | GitHub ID                                             | Affiliation |
+| ---------------- | ----------------------------------------------------- | ----------- |
+| Craig Perkins    | [cwperks](https://github.com/cwperks)                 | Amazon      |
+| Darshit Chanpura | [DarshitChanpura](https://github.com/DarshitChanpura) | Amazon      |
+| Peter Nied       | [peternied](https://github.com/peternied)             | Amazon      |
+
+### Processes
+
+* Approving Pull Requests, get two approval from the above contributors.  Expectations are lower that productionized code at current project kickoff, expectations to contribute into other branches follow their requirements.
+* Merge from main, this will be done weekly, @peternied to own
+* Merge into feature/extension, this should be done on demand as extension specific functionality is required. Handled on a case by case basis.  
+
+### End of life
```

Comment:
I'm not crazy about the use of end of life here. End of life implies the end of support for updates. Perhaps `End of feature branch` or maybe `Merge with main`?

`This branch will no longer be used for development in favor of development directly on main when the relevant...`